package.json

```diff
@@ -44,14 +47,17 @@
     "@backstage/e2e-test-utils": "^0.1.0",
     "@playwright/test": "^1.32.3",
     "@spotify/prettier-config": "^7.0.0",
-    "@types/node": "^18.0.0",
+    "@types/node": "^20.10.5",
     "concurrently": "^6.0.0",
-    "eslint-plugin-jest": "*",
+    "eslint": "^8.6.0",
+    "fs-extra": "10.1.0",
     "husky": "^8.0.0",
+    "jest": "^29.7.0",
+    "knip": "^3.9.0",
     "lint-staged": "^15.0.0",
     "node-fetch": "^2.6.7",
     "prettier": "^2.3.2",
-    "typescript": "~5.2.0"
+    "typescript": "^5.3.3"
```

Comment:
ts actually breaks stuff between minor versions, so `~` can be good.

Reply:
Good call, updated